src/PointwiseFunctions/AnalyticSolutions/GrMhd/AlfvenWave.hpp

```diff
@@ -124,7 +124,7 @@ class AlfvenWave {
   /// The background static magnetic field vector.
   struct BackgroundMagneticField {
     using type = std::array<double, 3>;
-    static std::string name() noexcept { return "BkgdMagneticField"; }
+    static std::string option_name() noexcept { return "BkgdMagneticField"; }
```

Comment:
We discussed this at some point and I think agreed to change this name, perhaps you were not present at that discussion. If I remember correctly, the reason was to allow for different names for the DataBox tag and the option. Once options can be retrieved through the DataBox (with PDALs) this may not be necessary anymore. Thoughts @wthrowe @nilsdeppe ?

Reply:
It is easy to get consensus if you don't invite the people who dislike the idea to the discussion.  I recall dropping this from one of your previous PRs because people (mainly me) didn't like it.

As far as I know, the only purpose for the DataBox names is observation, and you shouldn't be observing your input file options so it is not clear to me why we should support separate names.

Reply:
My view is that regardless of what consensus there is on the name, the change shouldn't be made as part of this PR. There seems to be a habit in these series of PRs to renames things. The renames should be presented in a separate PR where it is easy for anyone who has an opinion to join into the discussion.

Reply:
@wthrowe You make it sound as if I try to sneak this change in :D That is obviously not the case. I remembered we meant to do this change, so made it while touching these files. Don‘t care much about it, since as I said it‘s not so relevant anymore once we have PDALs. Please don’t suggest I intentionally exclude you from discussions to get my will.

Reply:
You're right, my comment was inappropriate.  I'm sorry.